src/ccl/ccl.template.html

```diff
@@ -96,18 +60,7 @@ <h2 class="h4">Temperature</h2>
           </div>
         </nav>
         <div class="charts scrollable">
-          <div class="chart">
-            <div class="chart-heading">
-              <h2 class="chart-label">
-                Total Precipitation
-              </h2>
-              <div class="chart-actions">
-                <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Export"><i class="icon icon-export"></i></button>
-                <button class="button button-link" data-toggle="tooltip" data-placement="bottom" data-delay="300" title="" data-original-title="Options"><i class="icon icon-cog"></i></button>
-              </div>
-            </div>
-            <div class="chart-body"></div>
-          </div>
+          <charts></charts>
```

Comment:
Does everything (styling, etc) still work if you don't nest this under the div, and instead do `<charts class="charts scrollable"></charts>`? Would clean up the template a bit and remove an unnecessary div.